src/components/data_display/routes.ts

```diff
@@ -149,11 +173,12 @@ const routeDataViewDetail = async (request: any, reply: any) => {
     linkedApplications as LinkedApplication[]
   )
 
+  if (returnRawData) return reply.send(response.item)
```

Comment:
The main use case I can think of is when a form wants to display data for a single specific item (e.g. additional "user" details, product info). You could just query the "table" endpoint and add some search filtering, but this makes it a bit more straightforward, and it was trivial to implement, so may as well.

Reply:
Oh and one other thing -- the detailView response usually has a lot more columns (the table view is limited to what can fit reasonably on the screen), so this might be desired to save having to make a separate `rawDataColumns` config field.